src/routes/github/webhook/webhook-receiver-post.ts

```diff
@@ -24,7 +26,8 @@ export const WebhookReceiverPost = async (request: Request, response: Response):
 			return;
 		}
 		webhookSecret = gitHubServerApp.webhookSecret;
-		const verification = createHash(JSON.stringify(payload), webhookSecret);
+		if (!webhookSecret) throw new Error("Cannot find webhookSecret for gitHubServerApp uuid: " + uuid);
```

Comment:
Actually, is empty webhook secret one of the use cases?
If so, we need another PR to bypass the `createhash` and `verification` 
If on, all good, leave it.

Reply:
Well, as DB allow null is false, so we don't have to worry webhookSecret is empty.